### PR TITLE
polish openshift-master role

### PR DESCRIPTION
**Pull request by ingvagabund**
Let's make the code more readable and closer to "best practices" of writing the Ansible.

### CODE REVIEW

File: roles/openshift_master/tasks/main.yml [hunk 1]
```diff
@@ -57,39 +67,43 @@
   args:
     creates: "{{ openshift_master_policy }}"
   notify:
-    - restart master
```
Comment (ingvagabund) [inline roles/openshift_master/tasks/main.yml]: Always confused by the indentation. This looks ugly to me but maybe it should be this way. Any justification why beside readability?

File: roles/openshift_master/tasks/main.yml [hunk 2]
```diff
@@ -125,20 +145,23 @@
...
 
 # This is an ugly hack to verify settings are in a file without modifying them with lineinfile.
 # The template file will stomp any other settings made.
 - block:
-    - name: check whether our docker-registry setting exists in the env file
-      command: "awk '/^OPENSHIFT_DEFAULT_REGISTRY=docker-registry.default.svc:5000/' /etc/sysconfig/{{ openshift.common.service_type }}-master"
-      failed_when: false
-      changed_when: false
-      register: already_set
+  - name: check whether our docker-registry setting exists in the env file
```
Comment (ingvagabund) [inline roles/openshift_master/tasks/main.yml]: @tbielawa undoing your change. Even if the Ansible docs shows as correct, I don't like it.

[1] http://docs.ansible.com/ansible/latest/playbooks_blocks.html#id1

Comment (ingvagabund) [inline roles/openshift_master/tasks/main.yml]: No matter which style we choose I would like to have that documented here [1]. Including indentation in both `block` and `notify`.

[1] https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by smarterclayton** [comment]
Can you layer this after https://github.com/openshift/openshift-ansible/pull/4832 (which removes a significant chunk of code, some of which you touch)?

File: roles/openshift_master/handlers/main.yml [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
   notify: Verify API Server
 
 - name: restart master api
-  systemd: name={{ openshift.common.service_type }}-master-api state=restarted
-  when: (openshift.master.ha is defined and openshift.master.ha | bool) and (not (master_api_service_status_changed | default(false) | bool)) and openshift.master.cluster_method == 'native'
+  systemd:
+    name: "{{ openshift.common.service_type }}-master-api"
+    state: restarted
+  when:
+  - openshift.master.ha is defined and openshift.master.ha | bool
```
Comment (tbielawa) [inline roles/openshift_master/handlers/main.yml]: I :heart: the changes to `AND`ing in list format

File: roles/openshift_master/tasks/main.yml [hunk 4]
```diff
@@ -57,39 +67,43 @@
...
 - name: Create the scheduler config
   copy:
     content: "{{ scheduler_config | to_nice_json }}"
     dest: "{{ openshift_master_scheduler_conf }}"
     backup: true
   notify:
-    - restart master
-    - restart master api
-    - restart master controllers
+  - restart master
```
Comment (tbielawa) [inline roles/openshift_master/tasks/main.yml]: :+1: for consistency in indentation

File: roles/openshift_master/tasks/systemd_units.yml [hunk 5]
```diff
@@ -12,65 +12,79 @@
...
   - not openshift.common.is_master_system_container | bool
   with_items:
   - api
   - controllers
-  register: create_ha_unit_files
+  register: l_create_ha_unit_files
 
 - command: systemctl daemon-reload
-  when: create_ha_unit_files | changed
+  when: l_create_ha_unit_files | changed
```
Comment (tbielawa) [inline roles/openshift_master/tasks/systemd_units.yml]: :+1: for fixing the variable names to match convention!

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
@smarterclayton so you know. @tbielawa can you re-review the PR one more time?

**Comment by ingvagabund** [comment]
[test]

**Comment by sdodson** [comment]
[merge]

**Pull request merged**